playbooks/sample-sap-hypervisor-redhat_ocp_virt-preconfigure.yml

```diff
@@ -4,8 +4,5 @@
   serial: 1
   vars:
     sap_hypervisor_node_platform: redhat_ocp_virt
-
-  tasks:
-    - name: Include Role
-      ansible.builtin.include_role:
-        name: sap_hypervisor_node_preconfigure
+  roles:
```

Comment:
this is not correct syntax 

see https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_reuse_roles.html#using-roles-at-the-play-level

Reply:
Well, thats actually what this PR should address. And yes, it was accidentally (auto suggested) created agains main, closing.